includes/API/Site/Controllers/SetupService.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Initializes all settings-related REST controllers for the Ad Partner plugin.
+ *
+ * This class acts as the central router for registering plugin-specific REST endpoints,
+ * primarily under the `wc/sfw/<ad_partner>` namespace.
+ *
+ * Controllers are instantiated with their required dependencies here and hooked into the REST API lifecycle.
+ *
+ * @package SnapchatForWooCommerce\API\Site\Controllers
+ */
+
+namespace SnapchatForWooCommerce\API\Site\Controllers;
+
+use Automattic\Jetpack\Connection\Manager;
+use SnapchatForWooCommerce\ServiceContainer;
+use SnapchatForWooCommerce\ServiceKey;
+use SnapchatForWooCommerce\Config;
+
+/**
+ * Bootstrap class for registering REST API routes related to plugin settings.
+ *
+ * Instantiates individual controllers and registers their routes.
+ *
+ * @since 0.1.0
+ */
+class SetupService {
```

Comment:
Nitpick: This could probably be moved up a level under includes/API/ so this directory is only where the controllers themselves live.